README.md

```diff
@@ -50,3 +50,32 @@ The following script can be used to build a host image for Google Compute Engine
 
 [Check out the AOSP tree](https://source.android.com/setup/build/downloading)
 to obtain the script.
+
+## Docker
+
+We also provide the docker image which installed cuttlefish debian packages
+inside; including `cuttlefish-base`, `cuttlefish-user`, and
+`cuttlefish-orchestration`.
+
+### Build docker image manually
+
+Please run below command to build manually.
```

Comment:
is it worth mentioning that it should run in arm machine?

Reply:
Yeah, I think so. Users can know with reading path of shell script, but not directly.

Additionally, what about this?
- Mentioning it should work in arm64.
- Changing the dir name from `docker/arm-server` to `docker/linux-env-builder-docker` and mentioning it should work in arm64 and x86_64 both. (PLMK if you have better naming suggestion.)